src/configuration.cc

```diff
@@ -156,8 +164,31 @@ int Configuration::ParseArguments(int ac, char** av) {
                 << std::endl;
       return -1;
     }
-
     ParseConfigFile(config_file);
+
+    // Command line options override the options provided in the config file.
+    if (flags.count("option")) {
+      std::stringstream input_stream;
```

Comment:
s/`input_stream`/`option_stream`/ — otherwise it looks weird that you're writing to it.

Reply:
done